TrailheadApex11/ExercisesPage/src/classes/Ex2CountingNumbers.cls

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by Kuba on 29.11.2017.
+ */
+
+public with sharing class Ex2CountingNumbers implements ExercisesLoader {
+    public String solveTheExercise(List<String> input) {
+        try {
+            String inputString = ExercisesController.isInputEmpty(input).get(0);
```

Comment:
No właśnie, przez to wrzucenie try/catchów powtarzasz kod cały czas.

DRY, DRY, DRY! Nie płacą nam za ilość kodu tylko za bycie sprytnym ;)
No i im mniej kodu do utrzymania tym lepiej.